Development-Guidelines/building-angular-project.md

```diff
@@ -16,18 +16,18 @@ Umbraco 7 needs a couple of things to run:
 ### Node.js 
 To compile and run the UI project you need Node.js installed, you can get that at [https://nodejs.org](nodejs.org) for both Windows and OSX.
 
-### Grunt
-When you have Node.js installed, you need to install Grunt. Grunt is a simple JavaScript task runner, basically like NAnt, MSBuild or any traditional build system [https://gruntjs.com](more about grunt here).
+### gulp
+When you have Node.js installed, you need to install gulp. gulp is a simple JavaScript task runner, basically like NAnt, MSBuild or any traditional build system [https://gulpjs.com](more about gulp here).
```

Comment:
[https://gulpjs.com](more about gulp here)
should be 
[more about gulp here](https://gulpjs.com), don't know why that was never caught in the original file 😄

Reply:
Would also be great if you capitalized Gulp - if nowhere else then in the title and after a period!

Reply:
Hiya thanks for taking a look.  I've fixed the references to the URL link.  Thanks for spotting it!

In terms of capitalisation - I decided to keep the lower case as this is how it seems to be referred to in their own documentation.  Similarly on the wiki regarding gulp is uses lower case g even within titles whereas the g in Grunt is capitalised.  So I concluded that gulp should always be lower case g.  However if you disagree then I would be happy to amend it as it is not something I'm going to be too precious about.

In terms of the commands - I did check the syntax from the gulp documentation.  It does indeed seem extremely similar.  However really appreciate it if you are going to test it within the context of Umbraco.  I did run the commands myself before I requested the PR but it always helps to have things verified :)

Once again - thanks.  Almost there!